openpype/hosts/flame/api/plugin.py

```diff
@@ -749,10 +765,37 @@ def _generate_media_info_file(self):
         # execute creation of clip xml template data
         try:
             openpype.run_subprocess(cmd_args)
+            self._make_single_clip_media_info()
         except TypeError:
             self.log.error("Error creating self.tmp_file")
             six.reraise(*sys.exc_info())
 
+    def _make_single_clip_media_info(self):
+        with open(self.tmp_file) as f:
+            lines = f.readlines()
+            _added_root = itertools.chain(
+                "<root>", deepcopy(lines)[1:], "</root>")
+            new_root = ET.fromstringlist(_added_root)
+
+        # find the clip which is matching to my input name
+        xml_clips = new_root.findall("clip")
+        matching_clip = None
+        for xml_clip in xml_clips:
+            if xml_clip.find("name").text in self.feed_basename:
+                matching_clip = xml_clip
+
+        if matching_clip is None:
+            # return warning there is missing clip
+            raise ET.ParseError(
```

Comment:
I'm not sure if it's good idea to raise `ParseError` from `ElementTree` because the xml was parsed.